src/components/FamilyTree/FamilyTree.jsx

```diff
@@ -55,32 +58,32 @@ function FamilyTree() {
       const dimensions = treeContainer.current.getBoundingClientRect();
       setState({
         translate: {
-          x: 30,
-          y: dimensions.height / 2
+          x: dimensions.width / 2,
+          y: 30
         }
       });
     }, []);
 
     /* Returns the actual tree content */
       /* Node shape */
-        const nodeShape = {
-          shape: 'rect',
-          shapeProps: {
-              width: 20,
-              height: 20,
-              x: -10,
-              y: -10,
-              stroke: 'steelblue',
-              strokeWidth: 3,
-              fill: 'white',
-          }
+      const nodeShape = {
+        shape: 'image',
+        shapeProps: {
+            width: 60,
+            height: 60,
+            x: -30,
+            y: -30,
+            href:"https://i.ibb.co/1n5F04c/tree-node.png", // Local links don't work somehow
```

Comment:
hmm thats weird since this is jsx did you try to import as a file name at the top and call it here as {img}

Reply:
yeah I tried that as a png and svg and it did not work. Rendered as a broken image. Attempted replacing that with the facebook icon that rendered and didn't work either lma